src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -49,6 +49,12 @@ function dd(mixed ...$vars): never
             header('HTTP/1.1 500 Internal Server Error');
         }
 
+        if (empty($vars)) {
+            VarDumper::dump('Dumped!');
```

Comment:
Should we use 🐛 like in the `dump()` function?

Reply:
we should be consistent with what we do in dump:
```php
VarDumper::dump(new ScalarStub('🐛'))
```